### ISSUE TITLE
Button ui style 

### ISSUE THREAD

**Issue by skanarul8002** [skanarul8002/mca_hub#13]
You choose any section and styling buttons as responsive

**Comment by jahnvi31** [comment]
can you assign this issue to me under hacktoberfest?

**Comment by skanarul8002** [comment]
yeah @jahnvi31 , please go through readme file first and full fill all criteria.
